src/ops.jl

```diff
@@ -4,18 +4,18 @@ import Base
 
 export extinct, extinct!
 
-DUST_LAWS = [:ccm89, :cal00, :od94]
+const DUST_LAWS = [ccm89, cal00, od94]
```

Comment:
Is this really necessary?  Users won't be able to pass in custom laws.

Reply:
ah, I see. This is better. Would this require users to import DustExtinction.jl, though? IE
```julia
using Spectra
spec = Spectrum(...)
extinct(spec, 0.4, 3.1, law=od94) # od94 is not defined, yet, so error?
...
using DustExtinction
extinct(spec, 0.4, 3.1, law=od94) # Now it works
```

Reply:
Yes, just like you did in the tests.  If you want, you can re-export those functions from in this package, so that they're readily available without manually loading `DustExtinction`

Reply:
There is a catch-22,though. If the we assert the law must exist in that const array, we're explicitly defining available laws, but by letting users pass laws as functions we're implying "if it looks like a law, talks like a law, etc." then it should pass. 

In general, I think it is more Julian to take the latter approach- let the user give any function with signature `law(wave, Rv)`. What do you think?

Reply:
I prefer accepting any law, so that users can plug their own laws.  Those in `DustExtinction` can just be the pre-defined ones.  Can I remove this constant and the assert below?

Reply:
Yes, go for it. I'll accept #1 as well so pull that down, if needed.

Reply:
Updated